src/device_info.sh

```diff
@@ -129,9 +153,17 @@ function get_cpu()
     return 0
   fi
 
-  cpu_currently=$(printf '%s\n' "$cpu_frequency" | grep 'CPU MHz')
-  cpu_max=$(printf '%s\n' "$cpu_frequency" | grep 'CPU max MHz')
-  cpu_min=$(printf '%s\n' "$cpu_frequency" | grep 'CPU min MHz')
+  cmd="printf '%s\n' '${cpu_frequency}' | grep 'CPU MHz'"
+  show_verbose "$flag" "$cmd"
+  cpu_currently=$(cmd_manager 'SILENT' "$cmd")
+
+  cmd="printf '%s\n' '${cpu_frequency}' | grep 'CPU max MHz'"
+  show_verbose "$flag" "$cmd"
+  cpu_max=$(cmd_manager 'SILENT' "$cmd")
+
+  cmd="printf '%s\n' '${cpu_frequency}' | grep 'CPU min MHz'"
+  show_verbose "$flag" "$cmd"
+  cpu_max=$(cmd_manager 'SILENT' "$cmd")
```

Comment:
Think that it should be `cpu_min` here. This added a bug that made the max frequency displayed to have the min frequency value and, also, made the min frequency to not be displayed.

Reply:
Nice catch!